src/SleepRepository.js

```diff
@@ -68,7 +68,7 @@ class SleepRepository {
   addSleepInput(input) {
     let foundInRecord = this.individualEntryRecords.find(record => record.date === input.date);
     if (foundInRecord) {
-      foundInRecord.hoursSlept = foundInRecord.numOunces + input.numOunces;
+      foundInRecord.hoursSlept = foundInRecord.hoursSlept + input.hoursSlept;
```

Comment:
Glad you caught this error! Thought this issue was going to be much more complex.